### DISCUSSION TITLE
i have issu with forge create

### DISCUSSION THREAD

**Discussion by alihaq27** [Cyfrin/foundry-full-course-cu#145] [Q&A] [answered]
hi i have issu whith this command in my terminal

forge create SimpleStorage --rpc-url http://127.0.0.1:7545
 --interactive
when i want to run this command i got this error:

forge create SimpleStorage --rpc-url http://127.0.0.1:7545
--interactive
[⠢] Compiling...
[⠆] Compiling 1 files with 0.8.19
[⠰] Solc 0.8.19 finished in 81.31ms
Compiler run successful!
Error: 
Error accessing local wallet. Did you set a private key, mnemonic or keystore?
Run `cast send --help` or `forge create --help` and use the corresponding CLI
flag to set your key via:
--private-key, --mnemonic-path, --aws, --interactive, --trezor or --ledger.
Alternatively, if you're using a local node with unlocked accounts,
use the --unlocked flag and either set the `ETH_FROM` environment variable to the address
of the unlocked account you want to use, or provide the --from flag with the address directly.
zsh: command not found: --interactive

i use ganache 
how can i fix this? please help

**Comment by wannabeOreki** [comment, marked as answer]
Use the below command instead (Note: here I am assuming you have successfully created a .env file and pasted your GANACHE_PRIVATE_KEY in it, if not then checkout this https://github.com/Cyfrin/foundry-full-course-f23/discussions/107 and [Private Key Rant II](https://youtu.be/umepbfKp5rI?t=27701) )

```bash
forge create SimpleStorage --rpc-url http://127.0.0.1:7545/ --private-key $GANACHE_PRIVATE_KEY
```
